client/components/themes-list/docs/example.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ThemesList from 'components/themes-list';
+
+const onScreenshotClickFn = function( theme, index ) {
+	console.log( `Theme ${ theme.id } at ${ index } clicked` );
+};
+
+const ThemesListExample = React.createClass( {
+	displayName: 'ThemesList',
+
+	render: function() {
+		const demoThemes = [ {
```

Comment:
Like `onScreenshotClickFn`, `demoThemes` can benefit from being placed up in the module scope.